### ISSUE TITLE
RRROBOTS (Reproducible Research in R for Ocean Biosciences: Open-science Training Seminar)

### ISSUE THREAD

**Issue by ha0ye** [mozillascience/WOW-2017#20]
Please follow this format for your issue. Look at the TEMPLATE issue if you have questions: https://github.com/mozillascience/WOW-2017/issues/3)

## PROJECT

TITLE: Reproducible Research in R for Ocean Biosciences: Open-science Training Seminar
PROJECT REPO: https://github.com/Open-Data-Science-at-SIO/RRROBOTS
DESCRIPTION: A repo for a class on reproducible research and data analysis training for graduate students.

## LINKS
- [x] README.md: https://github.com/Open-Data-Science-at-SIO/RRROBOTS/blob/master/README.md
- [x] Code of Conduct: https://open-data-science-at-sio.github.io/mission.html
- [x] Roadmap: https://github.com/Open-Data-Science-at-SIO/RRROBOTS/issues/2 [WIP]
- [x] CONTRIBUTING.md: https://github.com/Open-Data-Science-at-SIO/RRROBOTS/blob/master/CONTRIBUTING.md
- [x] LICENSE: https://github.com/Open-Data-Science-at-SIO/RRROBOTS/blob/master/LICENSE
- [x] Personas: https://github.com/Open-Data-Science-at-SIO/RRROBOTS

**Comment by auremoser** [comment]
Yay @ha0ye! Thank you so much for starting your issue and loading in links!

*githug*

![](http://i.giphy.com/42YlR8u9gV5Cw.gif)

Also, if you took any awesome photos this weekend, would it be alright to send you the dropbox to share a few of them? Your camera was way better than my iphone.

**Comment by ha0ye** [comment]
@auremoser Yep, happy to share. Feel free to send a link via email or twitter DM.
